addon/components/frost-link.js

```diff
@@ -309,6 +325,18 @@ export default LinkComponent.extend(PropTypeMixin, HookMixin, SpreadMixin, {
   },
 
   /**
+   * Open a link for an external location.
+   */
+  _openExternalLink () {
+    const href = this.get('href')
+    // If we want to open a new tab, the target will be set accordingly when we setup the routing.
+    // If we want to stay in the same tab, we will set the href.
+    if (this._shouldOpenInSameTab()) {
+      window.location.href = href
```

Comment:
I wasn't suggesting adding the ability to open a primary link in the same tab, but a secondary or tertiary link in a new tab. Just pointing out that without that functionality, `frost-link` is still unsusable for people who want/need to provide that functionality. 

Since I don't use `frost-link` now, I don't really care if you add this, just pointing out it doesn't solve the problem I raised about supporting external links in a way I can use.